diff --git a/share/metainfo/org.qt-project.qtcreator.appdata.xml.cmakein b/share/metainfo/org.qt-project.qtcreator.appdata.xml.cmakein
index b721029a0a..d6cbd34dfc 100644
--- a/share/metainfo/org.qt-project.qtcreator.appdata.xml.cmakein
+++ b/share/metainfo/org.qt-project.qtcreator.appdata.xml.cmakein
@@ -36,7 +36,7 @@
   </screenshot>
  </screenshots>
  <releases>
-  <release version="${IDE_VERSION_DISPLAY}"${DATE_ATTRIBUTE}>
+  <release version="${IDE_VERSION_DISPLAY}" ${DATE_ATTRIBUTE}>
    <description>
     <p>Qt Creator v${IDE_VERSION_DISPLAY}</p>
    </description>
